src/napari_builtins/io/_read.py

```diff
@@ -531,6 +531,24 @@ def patched_new(cls, *args, **kwargs):
         Viewer.__init__ = original_init
 
 
+@contextmanager
+def _patch_run():
+    """Context manager to patch the run method of the viewer."""
+    from napari._qt import qt_event_loop
+
+    original_ipython_check = qt_event_loop._ipython_has_eventloop
+
+    def patched_ipython_check() -> bool:
+        # Do not call the original run method
```

Comment:
```suggestion
        """A patched ipython_check that always returns True.
        
        Drag and drop feature uses this to prevent running another event loop.
        """
```